package.json

```diff
@@ -226,13 +221,10 @@
     "rimraf": "5.0.5",
     "semver": "7.5.4",
     "sinon": "17.0.1",
-    "terser-webpack-plugin": "5.3.9",
     "ts-loader": "9.5.1",
```

Comment:
ts-loader is the typescript loader for webpack and can also be removed

Reply:
Done